detekt-config.yml

```diff
@@ -21,7 +21,6 @@ test-pattern: # Configure exclusions for test sources
 
 build:
   warningThreshold: 5
-  failThreshold: 10
```

Comment:
Why was this removed?

Reply:
On similar pace (as other checks) that violations are only reported but do not fail the build process.